project/MiMaFilters.scala

```diff
@@ -21,6 +21,10 @@ object MiMaFilters {
     ProblemFilters.exclude[DirectMissingMethodProblem]("scala.deriving.Mirror.fromTuple"),
 
     // Private to the compiler - needed for forward binary compatibility
-    ProblemFilters.exclude[MissingClassProblem]("scala.annotation.since")
+    ProblemFilters.exclude[MissingClassProblem]("scala.annotation.since"),
+
+    // Runtime-only classes that we don't check (user code is not supposed to explicitly use scala.runtime anyway)
+    ProblemFilters.exclude[Problem]("scala.runtime.Invoker"),
+    ProblemFilters.exclude[Problem]("scala.runtime.Invoker$")
```

Comment:
Missed opportunity for trailing comma, a feature which was just on my mind.

Reply:
opportunity state: taken :ballot_box_with_check: 
let's improve future diffs